marklogic-data-hub-central/ui/src/components/entities/mapping/source-navigation/source-navigation.tsx

```diff
@@ -13,25 +14,25 @@ const SourceNavigation: React.FC<Props> = (props) => {
 
   return (
     <span className={styles.navigate_source_uris}>
-      <Button
+      <HCButton
```

Comment:
Can we center-align the text labels in the left and right buttons in the mapping source navigation? @jbelonoj do you agree?
<img width="530" alt="nav-button-align" src="https://user-images.githubusercontent.com/477757/132906291-e4046807-f34b-41e8-9413-d9b14f89df78.png">

Reply:
Yes, agreed! @wooldridge Also, I think the background colors are slightly off: 
<img width="100" alt="Screen Shot 2021-09-10 at 2 07 02 PM" src="https://user-images.githubusercontent.com/57465459/132918186-98b3a350-4794-4ab6-ba68-81fa22955bf4.png">

Reply:
@wooldridge I added both fixes, The background, and the alignment. It looks like this 
![image](https://user-images.githubusercontent.com/88790827/133121316-7b68a1aa-6fd2-44fb-b0a6-d5f5585f3f5a.png)